### ISSUE TITLE
querySelectorAll in Firefox, syntax error

### ISSUE THREAD

**Issue by Koshmaar** [adobe-webplatform/css-shapes-polyfill#17]
Hi, I used code like it was specified in readme:

```
<p> <img class="shape" src="hubert-photo.png" /> Blablabla (...) blablabla </p>

.shape
{
   border-radius: 50%;  /* gives  circle */
   float: right;
   shape-outside: circle(50%);
   shape-margin: 1em;
}
```

Tried in Firefox 47, and Palemoon 27, both give error in console:
**"SyntaxError: An invalid or illegal string was specified"**  in line 1379 of the not-minified source:

```
        new StylePolyfill(function(rules) {
            rules.forEach(function(rule) {
                var els = document.querySelectorAll(rule.selector);     // <------- here
                for (var i = 0; i < els.length; i++)
                    els[i].setAttribute('data-' + rule.property, rule.value);
            });
```

By inspecting variables I see **els** is undefined. So querySelectorAll didn't return anything.

The **rule** variable contains:

> property: "shape-outside"
> selector: "*/.shape"
> value: "circle(50%)"

The querySelectorAll doesn't work with this selector syntax. Do you know why? Is that Firefox specific limitation? 

If I change **selector** to ".shape" then the js doesn't break, querySelectorAll returns proper node. I don't see the script working as intended but it might be due to many things.  

Anyway, is the removal of */ a good workardound?  Where is the */ added in the file? I can't find it.

**Comment by oslego** [comment]
Hi, @Koshmaar!

I'm unable to replicate that issue on the latest Firefox stable (Firefox 50). Looking at the demo files in the `/demos/` folder and adding in your snippet (and including `width` and `height` properties) I get the expected result of the polyfill working and no errors. 

It shouldn't be an issue, but please try with the latest version of stable Firefox and let me know if the issue occurs again. If so, please make a very simple demo and post it somewhere so I can have a look.

**Comment by Koshmaar** [comment]
Hmm, very strange - the issue dissapeared. I don't know what has changed.. rather nothing. I stopped working on this thing until yesterday, when I saw this thing working in Palemoon. In code nothing changed, but in the meantime the browser has been restarted few times, and I also changed x64 to x86 version. Seems strange that x64 would cause such issue. Anyway, problem solved for now, thank you for your help :)
